### PR TITLE
Wired Rechargers

### PR DESCRIPTION
**Pull request by Geevies**
* Mapped in three wired rechargers to the engineering sub-level storage. They work off cables instead of APCs.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

File: code/modules/power/wired_recharger.dm [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+	name = "wired recharger"
+	desc = "Useful for recharging electronic devices innawoods."
+	icon = 'icons/obj/recharger.dmi'
+	icon_state = "wired_recharger_off"
+	anchored = TRUE
+	use_power = TRUE
+	idle_power_usage = 6
+	active_power_usage = 45 KILOWATTS
+	pass_flags = PASSTABLE
+	var/charging_efficiency = 1.3 // Entropy. The charge put into the cell is multiplied by this
```
Comment (Arrow768) [inline code/modules/power/wired_recharger.dm:11]: This value must be smaller than 1

**Review by Arrow768 — CHANGES_REQUESTED** [review]

**Comment by NonQueueingMatt** [comment]
Not really a fan of more chargers being added around honestly but I guess we already have some in the library.

**Review by NonQueueingMatt — APPROVED** [review]

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]
Hold on. Most of this code is just copypasted from the charger item. This should definitely be a subtype.

**Comment by NonQueueingMatt** [comment]
Inactive.

**Pull request closed without merging**